src/Components/Blazor/testassets/Directory.Build.props

```diff
@@ -1,10 +0,0 @@
-<Project>
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
-
-  <ItemGroup>
-    <!-- Workaround missing entries in deps files. Still under investigation -->
```

Comment:
Why were we able to delete this file?

Reply:
See eng\targets\CSharp.ReferenceAssembly.props